packages/sdk/react-client/src/echo/useSpaces.ts

```diff
@@ -30,12 +31,15 @@ export const useSpace = (spaceKey?: PublicKeyLike) => {
  */
 export const useOrCreateFirstSpace = () => {
   const client = useClient();
-  const spaces = useSpaces();
+  const identity = useIdentity();
+  const spaces = useSpaces({ all: true });
```

Comment:
realize this isn't changed in this PR, but I think the options for useSpaces should be a filter rather than a one-off "all", e.g., `useSpaces({ filter: SpaceState.ANY })` (i.e., filter is either a state or a function, or by default is the READY filter).

Reply:
agreed